packages/@lwc/jest-transformer/src/index.js

```diff
@@ -116,6 +116,12 @@ module.exports = {
             ...(semver.lt(compilerVersion, '2.49.1') ? { enableLwcSpread: true } : {}),
         });
 
+        // Log compiler warnings, if any
+        if (warnings && warnings.length > 0) {
+            warnings.forEach((warning) => {
+                console.warn(`\x1b[33m[LWC Warn]\x1b[0m(${filePath}): ${warning.message}`);
```

Comment:
```suggestion
                console.warn(`\x1b[33m[LWC Warn]\x1b[0m(${filePath}): ${warning?.message ?? warning}`);
```

Just in case. I believe Rollup plugins can pass anything in as a warning (e.g. strings).